src/common/ConfirmationDialogue.spec.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen, waitFor } from '@testing-library/react'
+import ConfirmationDialogue from './ConfirmationDialogue'
+
+const handleCancel = () => {
+  // dummy function
+}
+
+const handleConfirm = () => {
+  // dummy function
+}
+
+describe('ConfirmationDialogue', () => {
+  const testTitle = 'Test dialogue title'
+  const testText = 'Test dialogue text'
+
+  it('renders a ConfirmationDialogue component', async () => {
+    render(
+      <ConfirmationDialogue
+        showConfirmationDialogue
+        onCancel={handleCancel}
+        onConfirm={handleConfirm}
+        confirmationModalTitle={testTitle}
+        confirmationModalText={testText}
+      />
+    )
+
+    await waitFor(async () => {
+      expect(await screen.findByText(testTitle)).not.toBeUndefined()
+      expect(await screen.findByText(testText)).not.toBeUndefined()
+    })
+  })
+
+  it('does not render a ConfirmationDialogue component', async () => {
+    render(
+      <ConfirmationDialogue
+        showConfirmationDialogue={false}
+        onCancel={handleCancel}
```

Comment:
We should be able to replace these function calls with just:

onCancel={() => {}}